docs/content/users/quickstart.md

```diff
@@ -607,6 +607,65 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
     ddev launch /cp
     ```
 
+## Sulu
+
+```bash
+mkdir my-sulu-site && cd my-sulu-site
+ddev config --project-type=php --docroot=public
+ddev start
+ddev composer create sulu/skeleton
+```
+
+!!!tip "Multilingual"
+    To set up the administration interface for example in French alongside the default English, add the following snippet on top of `config/packages/sulu_admin.yaml` ([all available localizations](https://sulu.crowdin.com/sulusulu)):
+
+    ```bash
+    sulu_core:
+    locales:
+        en: English
+        fr: Francais
+    translations:
+        - en
+        - fr
+    ```
+
+    `ddev exec bin/console sulu:admin:download-language` downloads any set language. The only exceptions are English and German which are shipping with Sulu.
+
+The content management part of Sulu is built upon webspaces. Each webspace configures a content tree. Each content tree may contain translations for different locales. Create your default webspace configuration `mv config/webspaces/example.xml config/webspaces/my-sulu-site.xml` and adjust the values for `<name>` and `<key>` so that they are matching your project:
+
+```bash
+<?xml version="1.0" encoding="utf-8"?>
+<webspace xmlns="http://schemas.sulu.io/webspace/webspace"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.1.xsd">
+    <!-- See: http://docs.sulu.io/en/latest/book/webspaces.html how to configure your webspace-->
+
+    <name>My Sulu CMS</name>
+    <key>my-sulu-cms</key>
+```
+
+!!!warning "Caution"
+    Changing the `<key>` for a webspace later on causes problems. It is recommended to decide on the value for the key before the database is build in the next step.
+
+The information for the database connection is set in the environment variable `DATABASE:URL`. For development, clone the `.env` file by `cp .env .env.local` and change the environment variable to:
+
+```bash
+DATABASE_URL="mysql://db:db@db:3306/db?charset=utf8mb4"
+
+Now build the database. Building with the `dev` argument adds a user `admin`with the the password `admin` to your project.
+
+```bash
+ddev exec bin/adminconsole sulu:build dev
+ddev launch /admin
```

Comment:
what kind of Webserver is DDEV using do we need adjust something to support the media download and image generation. We have docs for [apache](https://docs.sulu.io/en/latest/cookbook/web-server/apache.html) and [nginx](https://docs.sulu.io/en/latest/cookbook/web-server/nginx.html).

Reply:
Beside the `var` directory which need to writeable in any Symfony application in dev. The `public/uploads` also need to writeable and in our case special vhost routing.

The Symfony CLI Webserver does not require special VHost the PHP Internal webserver requires a special routing file `php -S 127.0.0.1:8000 -t public config/router.php`.

Reply:
with ddev you are able to use apache as well as nginx, per default nginx is used. i've also pushed another commit to add `public/uploads` as the upload folder (are there more? - those are ignored by mutagen).

in regards of the other details you've raised i better hand over to @rfay and/or @stasadev, myself i am not a developer.

Reply:
and one other question @alexander-schranz in regards of multilingual. in the example i went with english in french to include one language aside the two sulu is shipping with. but with the snippet used in the quickstart i still get: 

```

Language: de
------------

 ! [NOTE] Download skipped because Sulu includes a translation for the language "de" per default.


Language: en
------------

 ! [NOTE] Download skipped because Sulu includes a translation for the language "en" per default.
```
when running `ddev exec bin/console sulu:admin:download-language` , why `de` is skipped? in the `sulu_admin.yaml` only english and french is set?

Reply:
`de` and `en` is shipped via `vendor/sulu/sulu` so there is nothing to download as only none core languages need to be download from https://sulu.crowdin.com/

Reply:
It looks to me like the default nginx config is working fine for displaying images:

<img width="972" alt="image" src="https://github.com/ddev/ddev/assets/112444/90576d9a-c360-4b87-b3eb-c5569830e232">